Package.swift

```diff
@@ -58,11 +58,11 @@ let package = Package(
         "GitHub",
         "Models",
         "Stripe",
-        .product(name: "Either", package: "swift-prelude"),
+        .product(name: "Either", package: "Prelude"),
         .product(name: "Logging", package: "swift-log"),
-        .product(name: "PostgreSQL", package: "postgresql"),
-        .product(name: "Prelude", package: "swift-prelude"),
-        .product(name: "Tagged", package: "swift-tagged"),
+        .product(name: "PostgreSQL", package: "PostgreSQL"),
```

Comment:
(And even if that _does_ fix it, we don't control this dependency, so we'd have to fork it just to get the naming right.)